pkg/systemd/overview-cards/shutdownStatus.jsx

```diff
@@ -0,0 +1,116 @@
+/*
+ * This file is part of Cockpit.
+ *
+ * Copyright (C) 2022 Red Hat, Inc.
+ *
+ * Cockpit is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation; either version 2.1 of the License, or
+ * (at your option) any later version.
+ *
+ * Cockpit is distributed in the hope that it will be useful, but
+ * WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with Cockpit; If not, see <http://www.gnu.org/licenses/>.
+ */
+import React, { useState, useEffect } from 'react';
+import { Button, Flex, FlexItem } from '@patternfly/react-core';
+import { PowerOffIcon, RedoIcon } from "@patternfly/react-icons";
+
+import * as timeformat from "timeformat";
+
+import cockpit from "cockpit";
+
+const _ = cockpit.gettext;
+
+const getScheduledShutdown = (setShutdownTime, setShutdownType) => {
+    const client = cockpit.dbus("org.freedesktop.login1");
+    return client.call("/org/freedesktop/login1", "org.freedesktop.DBus.Properties", "Get",
+                       ["org.freedesktop.login1.Manager", "ScheduledShutdown"], { type: "ss" })
+            .then(([result]) => {
+                if (result) {
+                    setShutdownType(result.v[0]);
+                    setShutdownTime(result.v[1]);
+                }
+            })
+            .catch(err => console.warn("Failed to get ScheduledShutdown property", err));
+};
+
+const cancelShutdownAction = () => {
+    const client = cockpit.dbus("org.freedesktop.login1");
+    return client.call("/org/freedesktop/login1", "org.freedesktop.login1.Manager", "CancelScheduledShutdown")
+            .then(reply => {
+                if (!reply[0]) {
+                    console.error("Unable to cancel shutdown");
+                }
+            })
+            .catch(err => console.warn("Failed to cancel shutdown", err));
+};
+
+export const ShutDownStatus = () => {
+    const [shutdownType, setShutdownType] = useState(null);
+    const [shutdownTime, setShutdownTime] = useState(0);
+
+    useEffect(() => {
+        getScheduledShutdown(setShutdownTime, setShutdownType);
+
+        if (shutdownType === null) {
+            // logind does not have a propertieschanged mechanism https://github.com/systemd/systemd/issues/22244
+            cockpit.file("/run/systemd/shutdown/scheduled").watch(content => {
```

Comment:
content is unused, `() =>` pelase; also, it's just a single call, so it can just be `=> handler()` without the {}